package.json

```diff
@@ -124,6 +124,7 @@
     "@babel/plugin-syntax-dynamic-import": "^7.0.0",
     "@babel/plugin-syntax-import-meta": "^7.0.0",
     "@babel/plugin-transform-flow-strip-types": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.4.3",
```

Comment:
Hmm! I am suspicious about this one! I've had issues with this in the past. Checking out the compiled files now.. 🤞

Reply:
@tinkertrain Ugh... yea. It's what I feared.

Check out the babel compiled code:

```
var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

exports.__esModule = true;
exports.default = void 0;

var _extends2 = _interopRequireDefault(require("@babel/runtime/helpers/extends"));

var _objectWithoutPropertiesLoose2 = _interopRequireDefault(require("@babel/runtime/helpers/objectWithoutPropertiesLoose"));
```

Prior to `@babel/runtime`, these functions were not dependencies.
With this set up.. that means that consuming integrations HAVE to also use `@babel/runtime` to compile things.

I remember running into these issues during the Brigade v1.x work.

Is there a way we can get Jest 24 set up, without `@babel/runtime`? 

😊

Reply:
I was afraid of something like that!

I have pushed a solution... feels a little bit like cheating...

Reply:
It's not cheating if it works! Haha